### ISSUE TITLE
[error] calling "trezorctl get_features" on Python 3.4.6

### ISSUE THREAD

**Issue by SimonVrouwe** [trezor/python-trezor#256]
calling ```trezorctl get_features``` gives error
```
Traceback (most recent call last):
  File "/usr/bin/trezorctl", line 851, in <module>
    cli()
  File "/usr/lib/python3.4/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.4/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.4/site-packages/click/core.py", line 1031, in _process_result
    **ctx.params)
  File "/usr/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/bin/trezorctl", line 99, in print_result
    click.echo(format_protobuf(res))
  File "/usr/lib/python3.4/site-packages/trezorlib/client.py", line 119, in format_protobuf
    return pb.__class__.__name__ + ' ' + pformat_value(pb.__dict__, indent)
  File "/usr/lib/python3.4/site-packages/trezorlib/client.py", line 111, in pformat_value
    lines.append(leadin + key + ': ' + pformat_value(val, indent + 1) + ',')
  File "/usr/lib/python3.4/site-packages/trezorlib/client.py", line 115, in pformat_value
    return 'bytearray(0x{})'.format(value.hex())
AttributeError: 'bytearray' object has no attribute 'hex'
```
using ```trezorctl version
0.9.1``` with ```Python 3.4.6``` on opensuse 42.3, I think ```trezor``` was installed using pip3.

On another machine with ```Python 3.6.4``` it works fine.

**Comment by matejcik** [comment]
this is already fixed in master, but yes, it seems this bug has slipped into the released 0.9.1 version

**Comment by matejcik** [comment]
fix released
